pf4j/src/main/java/org/pf4j/PluginClassLoader.java

```diff
@@ -57,11 +58,19 @@ public PluginClassLoader(PluginManager pluginManager, PluginDescriptor pluginDes
      * before trying to load the a class through this loader.
      */
     public PluginClassLoader(PluginManager pluginManager, PluginDescriptor pluginDescriptor, ClassLoader parent, boolean parentFirst) {
+        this(pluginManager, pluginDescriptor, parent, parentFirst ? DefaultClassLoadingStrategy.PARENT_FIRST: DefaultClassLoadingStrategy.DEFAULT_ORDER);
+    }
+
+
+    /**
+     * classloading according to {@code classLoadingOrder}
+     */
+    public PluginClassLoader(PluginManager pluginManager, PluginDescriptor pluginDescriptor, ClassLoader parent, Set<ClassLoadingSource> classLoadingOrder) {
```

Comment:
The same here, pass a `ClassLoadingStrategy`

Reply:
done